### ISSUE TITLE
Research and setup CI/CD Pipeline

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#11]
## User story
1. As a Team
2. We want readable, uniform and well tested code
3. So that our codebase stays manageable during the whole project

## Acceptance criteria
* Research how to use [GitHub Actions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) to build a CI/CD Pipeline
* Create a dummy pipeline that runs on every push onto every branch
* Research about how to invoke the linter from the pipeline

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
